public/media-hunt/.env.example

```diff
@@ -0,0 +1 @@
+TMDB_API_TOKEN=eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJmMzEwZDg2NTM4OTkzYmNhMjNlNWUyOWNkYjk3NGFlNCIsInN1YiI6IjVjMWM5OTQwOTI1MTQxMjdmODRlMWVmNSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.csMyKzkblGa8r-2EPryAbFrm_k5LcOZae1tgvTAXxFY
```

Comment:
Please remove value from example, only the key needs to be commited.

Reply:
removed